### PR TITLE
Publicize: new notification setting

### PR DESCRIPTION
**Pull request by artpi**
New notification setting for publicize scheduled action.

D6090-code introduces notifications for advanced social media functionality.
Some users can have loads of activity, so this setting 

Backend work for this has been done in D6090-code .

zrzut ekranu 2017-06-23 o 15 23 48

## How to test?

1. Apply D6090-code
2. go to http://calypso.localhost:3000/me/notifications , find your sites, disable notifications for publicize.
3. Go to wpsh, follow instructions for D6090-code (replace user id and blog id with yours)
4. See that notification did not come, even after a reload
5. Exit out of wpsh. That is importand because caching
6. Reenable notifiactions for publicize on your site with http://calypso.localhost:3000/me/notifications
6. Repeat step 3
7. You should see notification (after a while or after a reload)

### CODE REVIEW

**Comment by sirbrillig** [comment]
Followed instructions as written, but it didn't work for me. I didn't see a notification no matter what I did, however, this may be due to my incorrect testing.

If I use the `wpsh` command as-written in the D6090, replacing just your userID with mine, I get a notification. Turning off the setting seems to have no effect on this.

Per these instructions, I also tried replacing the `blog_id` (and, since it seemed to be implied, the `post_id`) but then I was unable to get a notification at all, no matter the setting.

I think that there might be more implied in the testing instructions than was written. Maybe I also need to use my own `connection_id` or `url`? Also, the last argument is unclear. The instructions in D6090 read that it `is unique to make notifications not double-appear. use something different when testing` but that doesn't really explain what "different" thing I should use? Is it an ID? What should it be?

File: client/me/notification-settings/settings-form/locales.js [hunk 1]
```diff
@@ -13,7 +13,8 @@ export const settingLabels = {
 	post_like: () => i18n.translate( 'Likes on my posts' ),
 	follow: () => i18n.translate( 'Site follows' ),
 	achievement: () => i18n.translate( 'Site achievements' ),
-	mentions: () => i18n.translate( 'Username mentions' )
+	mentions: () => i18n.translate( 'Username mentions' ),
+	scheduled_publicize: () => i18n.translate( 'Post Publicized' )
```
Comment (sirbrillig) [inline client/me/notification-settings/settings-form/locales.js]: Any reason not to use a trailing comma here?

File: client/me/notification-settings/blogs-settings/blog.jsx [hunk 2]
```diff
@@ -69,7 +69,15 @@ class BlogSettings extends Component {
...
 							} }
-							settingKeys={ [ 'new_comment', 'comment_like', 'post_like', 'follow', 'achievement', 'mentions' ] } />;
+							settingKeys={ [
+								'new_comment',
+								'comment_like',
+								'post_like',
+								'follow',
+								'achievement',
+								'mentions',
+								'scheduled_publicize'
```
Comment (sirbrillig) [inline client/me/notification-settings/blogs-settings/blog.jsx]: Any reason not to use a trailing comma here?

**Comment by artpi** [comment]
> Followed instructions as written, and it did work. However, I want to make sure I'm testing correctly.

I think you didn't exit wpsh, hence the setting you were changing because it is cached the moment you open wpsh :)
Took me a while to figure that one!

**Pull request merged**